package.json

```diff
@@ -21,7 +21,8 @@
     "vue-i18n": "^9.2.2",
     "vue-router": "^4.1.6",
     "vue-search-input": "^1.1.11",
-    "vue3-mq": "^3.1.3"
+    "vue3-mq": "^3.1.3",
+    "vue3-toggle-button": "^1.0.3"
```

Comment:
is this used for internationalization toggle, just curious

Reply:
That's actually for responsive design for screen dimension.